docs/common-bugs.rst

```diff
@@ -344,7 +344,7 @@ Automatic installation in Live OS
 
 :Issue: The reporter would like to run a kickstart installation in Live OS.
 :Solution: Kickstart installations in Live OS are not supported. Please, run the installation with
-    ``boot.iso``.
+    ``boot.iso`` ("netinst") for the Server edition of Fedora.
```

Comment:
I would try to expand the sentence a little more with something like:
`... or use a netinstall or standard ISO (for example, with the Server edition of Fedora).`

Reply:
Is netinstall something else than boot.iso?

Reply:
Resolved out of band: netinstall is a subset of boot.iso. Being a boot.iso means the media is built to boot into anaconda. netinstall then has only that, while the other (unnamed) possibility is to have also packages on the media.